---
id: 339
title: 'A Bioinformatics Study - Annotation of Drosophila Erecta'
date: 2009-11-10T19:38:14+00:00
author: arisamuel
type: posts
layout: single
<!-- guid: http://ariak.wordpress.com/?p=339 -->
permalink: /A-Bioinformatics-Study-Annotation-of-Drosophila-Erecta
ks_metadata:
  - 'a:7:{s:4:"lang";s:2:"en";s:8:"keywords";s:52:"gene,dere,fig,genes,dmel,predicted,drosophila,cg7140";s:19:"keywords_autoupdate";s:1:"1";s:11:"description";s:150:"gene location and sizes. Furthermore, three putative genes were predicted with varying degrees of evidence. But despite the general similarity in gene";s:22:"description_autoupdate";s:1:"1";s:5:"title";s:0:"";s:6:"robots";s:12:"index,follow";}'
kdc_metadata:
  - 'a:1:{s:4:"lang";s:2:"en";}'
gwo4wp:
  - 'a:4:{s:7:"enabled";s:0:"";s:14:"control_script";s:0:"";s:15:"tracking_script";s:0:"";s:17:"conversion_script";s:0:"";}'
  - 'a:4:{s:7:"enabled";s:0:"";s:14:"control_script";s:0:"";s:15:"tracking_script";s:0:"";s:17:"conversion_script";s:0:"";}'
original_post_id:
  - "339"
categories:
  - Science, Experiments and Technical Stuff
  - Uncategorized
tags:
  - bioinformatics
  - genetics
---
A Bioinformatics Study & ReportA paper I wrote for a bioinformatics class several years ago in which I annotated part of the Drosophila Erecta genome (Fosmid 14) as part of a concerted effort to annotate the entire genome. The study was spearheaded by Washington U. in Seattle. This paper was A Bioinformatics Study & Report written for a class I took at SFSU in 2007. A link to the pdf version is below.

<a href="http://www.diffusionreactor.com/resarch-analysis/a-bioinformatics-study-annotation-of-drosophila-erecta/attachment/aakerstein_final_bioinformatics_submission-copyforblog-4/" rel="attachment wp-att-349">Bioinformatics-Drosophila_Annotation</a>

<div>
  <p align="center">
    <strong>Annotating Drosophila Erecta, fosmid 14</strong>
  </p>
  
  <p>
    <strong>Abstract, </strong>
  </p>
  
  <p>
    Fosmid14 of Drosophila Erecta shows high similarity to Drosophila Melanogaster.Â  Fosmid analysis reveals 3 nearly identical orthologous genes with similar intron / exon patterns, gene location and sizes. Furthermore, three putative genes were predicted with varying degrees of evidence.
  </p>
  
  <p>
    But despite the general similarity in gene structure between the two species, there is some difference in the placement and density of transposable elements (TEs). Dere shows a higher density of TEs on both the forward and reverse strands. Of particular interest is a TE cluster upstream of Dere CG7140 in roughly the space one would expect an ortholog to Dmel CG7139 (absent on fosmid14). While perplexing, this may suggest the erosion of a gene (CG7139) between species with replacement by a cluster of TEs. Further, Clustal QT alignments and Ka/Ks analysis indicate strong purifying selection<em>.</em>
  </p>
  
  <p>
    <strong>Global structure, Dere fosmid14</strong>
  </p>
  
  <p>
    Dere fosmid 14 from GEP was uploaded in .xml format to Apollo, a gene curation program. The GEP project report indicated there were 6 predicted genes on Dere Fosmid14. The GEP project report estimated 3 of these to be real. Repeat masker predicted a repeat density of 2.42%.
  </p>
  
  <p>
    Using Flybase D. melanogaster sequence as reference, several blasts were run (blastp, blastn, tblastn) against each gene &#8211; both total sequence blasts as well as exon by exon &#8211; to validate these predictions. It was concluded that the three orthologous genes were properly predicted while only one of the three genes predicted by gene predictor programs (Genscan, SNAP) were correct.
  </p>
  
  <p>
    The overarching assumption in this work is that annotation of D. Erecta should be done in comparison to its better-understood relative, Drosophila Melanogaster. Indeed, one expects the two genetic profiles to be nearly identical. This raises the somewhat philosophical question of how many / which (types) of genes ultimately distinguish one species from another. Perhaps it is not the gene structure, per se, but other features such as repeats or transposable elements that account for variety between close species. Regardless, one expects to find inevitable differences between genomes, but that they should remain generally similar. Features considered most notable are deviations in genomic data as this indicates features that, at least in part, distinguish one species from another. How exactly such differences contribute to speciation remains a puzzle. Suffice it to say that the annotators alterations to the fosmid were noted and accounted for to the greatest extent possible. What follows is a discussion about the features considered the most interesting on the fosmid.
  </p>
  
  <p>
    <strong><em><a href="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig1.png"><img title="Apollo - Image1" src="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig1-300x218.png?resize=300%2C218" alt="" data-recalc-dims="1" /></a>Â </em></strong>
  </p>
  
  <p>
    <strong><em>Fig 1. Global structure of Drosophila Erecta, fosmid 14, (including putative genes)</em></strong>
  </p>
  
  <p>
    <strong>Â </strong>
  </p>
  
  <p>
    <strong>Gene by Gene</strong><strong> analysis:</strong>
  </p>
  
  <p>
    <strong>TyrR</strong>
  </p>
  
  <p>
    The molecular function of TyrR is described as being involved in smell perception, muscle contraction, nerve-nerve synaptic transmission, G-protein coupled receptor activity. There is a possible phenotype association with neuro-muscular junctions. Blast results reveal 97% similarity and 96% identity. The number of exons and introns is preserved between the genes, as is a long, distinctive 5â€™ UTR. Splice sites are well-predicted between all the gene predictor programs (SNAP, Genscan etc.). There are two TEs within the Dere first intron that are also found in Dmel. However these TEs were unannotated in Dmel. The third intron in Dere is slightly longer<strong>.Â  </strong>GleanR comparisons reveal this gene is well-conserved between Drosophilidae.
  </p>
  
  <p>
    <strong>Drosophila Melanogaster</strong>
  </p>
  
  <p>
    <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig2a.png"><img class="alignnone size-medium wp-image-800" title="fig2a" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig2a-300x75.png?resize=300%2C75" alt="" data-recalc-dims="1" /></a>
  </p>
  
  <p>
    <strong>Drosophila Erecta</strong>
  </p>
  
  <p>
    <a href="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig2b.png"><img class="alignnone size-medium wp-image-801" title="fig2b" src="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig2b-300x103.png?resize=300%2C103" alt="" data-recalc-dims="1" /></a>
  </p>
  
  <p>
    <strong><em>Fig 2. Side by side comparison between Dmel TyrR and Dere TyrR</em></strong>
  </p>
  
  <p>
    &nbsp;
  </p>
  
  <p>
    <em><a href="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig3a.png"><img class="alignnone size-medium wp-image-802" title="fig3a" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig3a-300x171.png?resize=300%2C171" alt="" data-recalc-dims="1" /></a></em>
  </p>
  
  <p>
    <a href="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig3b.png"><img class="alignnone size-medium wp-image-803" title="fig3b" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig3b-300x165.png?resize=300%2C165" alt="" data-recalc-dims="1" /></a>
  </p>
  
  <p>
    <em>Â <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig3c.png"><img class="alignnone size-medium wp-image-804" title="fig3c" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig3c-300x71.png?resize=300%2C71" alt="" data-recalc-dims="1" /></a></em>
  </p>
  
  <p>
    <strong><em>Fig 3. Clustal QT alignment for TyrR between Dere and other Drosophilidae</em></strong>
  </p>
  
  <p>
    <strong><em>Â </em></strong>
  </p>
  
  <p>
    <em>Â <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig4.png"><img class="alignnone size-medium wp-image-805" title="fig4" src="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig4-300x255.png?resize=300%2C255" alt="" data-recalc-dims="1" /></a></em>
  </p>
  
  <p>
    <strong><em>Fig.4. Tree view for Dere TyrR based on Clustal evidence. </em></strong>
  </p>
  
  <p>
    <em>Â </em>
  </p>
  
  <p>
    <em>Â <a href="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig5.png"><img class="alignnone size-medium wp-image-806" title="fig5" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig5-300x100.png?resize=300%2C100" alt="" data-recalc-dims="1" /></a></em>
  </p>
  
  <p>
    <strong><em>Fig 5. Ka/Ks analysis (Ka/Ks score = .0511) for TyrR indicates strong purifying selection</em></strong><em><br /> </em>
  </p>
  
  <p>
    <em><br /> </em>
  </p>
  
  <p>
    <strong>CG14561</strong>
  </p>
  
  <p>
    Blast analysis shows these genes to be 88% identical and 92% similar. This is a one-exon gene with well-predicted splice sites and size. UTR structure is only slightly altered between species. Molecular function remains unknown. This seems a straight-forward example of a gene that is well-preserved and well-predicted.
  </p>
  
  <p>
    <em>Â <a href="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig6.png"><img class="alignnone size-medium wp-image-807" title="fig6" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig6-300x96.png?resize=300%2C96" alt="" data-recalc-dims="1" /></a></em>
  </p>
  
  <p>
    <strong><em>Fig 6. Clustal alignment, CG14561 between Drosophilids</em></strong>
  </p>
  
  <p>
    <strong><em><br /> </em></strong>
  </p>
  
  <p>
    <strong><em>Drosophila Erecta</em></strong>
  </p>
  
  <p>
    <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig7a.png"><img class="alignnone size-medium wp-image-808" title="fig7a" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig7a-300x89.png?resize=300%2C89" alt="" data-recalc-dims="1" /></a>
  </p>
  
  <p>
    <strong><em>Drosophila Melanogaster</em></strong>
  </p>
  
  <p>
    <a href="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig7b.png"><img class="alignnone size-medium wp-image-809" title="fig7b" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig7b-300x164.png?resize=300%2C164" alt="" data-recalc-dims="1" /></a>
  </p>
  
  <p>
    <strong><em>Fig. 7. Comparison between Dmel CG14561 and Dere CG14561</em></strong>
  </p>
  
  <p>
    <strong><em><br /> </em></strong>
  </p>
  
  <p>
    <strong>CG7140</strong>
  </p>
  
  <p>
    CG7140 is 96% similar and 93% identical between species. Molecular function is described as glucose-6-phosphate 1-dehydrogenase activity. The biological function is in glucose and monosaccharide metabolism. Gene predictors (Genscan) falsely predict multiple exons. Blast analysis indicates this is however a single exon gene. Interestingly there is a high TE density upstream of CG7140 not seen in Dmel. cDNA evidence is present though splice sites arenâ€™t well defined. This gene was blasted against GleanR proteins and is well conserved between most Drosophilidae.
  </p>
  
  <p>
    <strong><em>Drosophila Erecta<br /> </em></strong>
  </p>
  
  <p>
    <strong><em><a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig8a.png"><img class="alignnone size-medium wp-image-810" title="fig8a" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig8a-300x238.png?resize=300%2C238" alt="" data-recalc-dims="1" /></a>Â </em></strong>
  </p>
  
  <p>
    <strong><em>Drosophila Melanogaster </em></strong>
  </p>
  
  <p>
    <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig8b.png"><img class="alignnone size-medium wp-image-811" title="fig8b" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig8b-300x97.png?resize=300%2C97" alt="" data-recalc-dims="1" /></a><strong><em></em></strong>
  </p>
  
  <p>
    <strong><em>FIg</em></strong><strong><em> 8. Comparison between CG7140 in Dere and Dmel, respectively<br /> </em></strong>
  </p>
  
  <p>
    <strong><em><br /> </em></strong>
  </p>
  
  <p>
    <a href="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig9.png"><img class="alignnone size-medium wp-image-812" title="fig9" src="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig9-300x151.png?resize=300%2C151" alt="" data-recalc-dims="1" /></a>
  </p>
  
  <p>
    <strong><em>Fig 9. Clustal alignment for CG7140 between some members of drosophilidae</em></strong>
  </p>
  
  <p>
    <strong><em>Â </em></strong>
  </p>
  
  <p>
    <strong><em>Â <a href="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig10.png"><img class="alignnone size-medium wp-image-813" title="fig10" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig10-300x145.png?resize=300%2C145" alt="" data-recalc-dims="1" /></a></em></strong>
  </p>
  
  <p>
    <strong><em>Fig</em></strong><strong><em> 10. Ka/Ks analysis (Ka/Ks score= .03) for CG7140 reveals strong purifying selection.</em></strong>
  </p>
  
  <p>
    <strong><em><br /> </em></strong>
  </p>
  
  <p>
    <strong><em>Â <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig11.png"><img class="alignnone size-medium wp-image-814" title="fig11" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig11-300x162.png?resize=300%2C162" alt="" data-recalc-dims="1" /></a></em></strong>
  </p>
  
  <p>
    <strong><em>Fig 11. Tree view for CG7140 based on Clustal data</em></strong>
  </p>
  
  <p>
    &nbsp;
  </p>
  
  <p>
    <strong>Fosmid</strong><strong> 14, Gene 2</strong>
  </p>
  
  <p>
    This was the second predicted gene on fosmid 14 and the only one that was kept. Initial evidence came from <em>Genscan</em><em>. </em>Blastp against all drosophilidae GleanR proteins were run on Flybase (<a href="http://www.flybase.org/">www.flybase.org</a>) with no hits found. Interproscan analysis however reveals some conserved functional domains (Fig. 5). Despite spotty cDNA evidence, the combination of gene prediction coupled with conserved protein domains seems enough to warrant keeping.
  </p>
  
  <p>
    <a href="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig12.png"><img class="alignnone size-medium wp-image-815" title="fig12" src="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig12-300x169.png?resize=300%2C169" alt="" data-recalc-dims="1" /></a>
  </p>
  
  <p>
    <strong><em>Fig</em></strong><strong><em> 12. Interproscan results for fosmid14, gene2, illustrating conserved protein domains despite a lack of supporting Blast data.</em></strong>
  </p>
  
  <p>
    What does it mean that there are no gleanR results against the near relatives and yet some conserved functional domains? This is a question without a good answer. In the absence of more information the gene is worth annotating.
  </p>
  
  <p>
    <strong>Other predicted genes: Fosmid14 gene1 and gene3</strong>
  </p>
  
  <p>
    While these predicted genes were eventually deleted, a brief discussion about the decision-making process is probably in order.Â  With respect to gene 3, if one compares the Dmel region just upstream of CG7140 and opposite TyrR it seems reasonable to think that fosmid14_gene3 might actually be dmel_CG7139. Working under this assumption several blasts were run, against dmel_CG7139, tblastn was run against dmel all translations and finally blastp against GleanR (Drosophilidae) was tested. All results indicated that indeed there is no correlation between the two genes. Fosmid14_gene3 nucleotide and peptide sequences were both run on interproscan to determine if any functional domains exist. Again no hits. This led to the conclusion that the region, for reasons unknown, was falsely predicted by Genscan. While a more detailed analysis of the Genscan algorithm might provide further insight it is beyond the scope of this paper.
  </p>
  
  <p>
    Fosmid14_gene1 followed a similar treatment as gene3, though in this case it seemed less likely from the outset that this was a gene, as the Dmel region 5â€™ of TyrR shows little activity (e.g. no repeats, TEs or genes). The results suggest that this too was a false-positive prediction.
  </p>
  
  <p>
    <strong><em>Â <a href="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig13.png"><img class="alignnone size-medium wp-image-816" title="fig13" src="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig13-300x161.png?resize=300%2C161" alt="" data-recalc-dims="1" /></a></em></strong>
  </p>
  
  <p>
    <strong>Fig 13. View of Predicted genes 1 and 3 (note: gene 5, bottom right is actually gene 3 misnamed)</strong>
  </p>
  
  <p>
    <strong><em>Transposable Element structure</em></strong>
  </p>
  
  <p>
    <strong><em>Â Â Â Â Â Â Â Â Â Â Â  </em></strong><em>Analysis of TE density in and around the previously mentioned genes reveals some differences between Dmel and Dere. By examining the relative density of TE regions one observes a higher frequency of TEs in Dere on both the forward and reverse strands. Perhaps the most interesting finding with respect to TEs is a density just upstream of fosmid14, gene3 (since determined to be a false positive gene prediction).</em>
  </p>
  
  <p>
    One could hypothesize that this repeat density is somehow correlated with Dmel CG7139, which is just upstream of CG7140. Might this be illustrative of the erosion of CG7139 as it is invaded by TEs? Might TEs simply manifest at the sites of gene erosion for other reasons? Such questions, while interesting, are beyond the scope of this paper.<em><br /> </em>
  </p>
  
  <p>
    <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig14.png"><img class="alignnone size-medium wp-image-817" title="fig14" src="https://i0.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig14-300x151.png?resize=300%2C151" alt="" data-recalc-dims="1" /></a>
  </p>
  
  <p>
    <strong><em>Fig 14. Reverse strand comparison between Dere and Dmel. A higher repeat density is easily observed on Dere surrounding CG7140 and (falsely) predicted fosmid14_gene3. One also observes a relationship between the TE structure flanking fosmid14_gene3 and the absence of CG7139 from Dere. </em></strong>
  </p>
  
  <p>
    <em>Â </em>
  </p>
  
  <p>
    <em>Â <a href="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig15.png"><img class="alignnone size-medium wp-image-818" title="fig15" src="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig15-227x300.png?resize=227%2C300" alt="" data-recalc-dims="1" /></a></em>
  </p>
  
  <p>
    <strong><em>Fig 15. Blastn results between the Wolbachia genome and Fosmid14 reveals no significant lateral or horizontal gene transfer. </em></strong>
  </p>
  
  <p>
    <em>Â </em>
  </p>
  
  <p>
    <strong>Â <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig16.png"><img class="alignnone size-medium wp-image-819" title="fig16" src="https://i1.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig16-300x226.png?resize=300%2C226" alt="" data-recalc-dims="1" /></a></strong>
  </p>
  
  <p>
    <strong><em>Table 16. Comparison between Dmel and Dere of relative TE densities in the regions flanking fosmid 14 genes</em><br /> </strong>
  </p>
  
  <p>
    <em>Â <a href="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig17.png"><img class="alignnone size-medium wp-image-820" title="fig17" src="https://i2.wp.com/www.diffusionreactor.com/wp-content/uploads/2009/11/fig17-289x300.png?resize=289%2C300" alt="" data-recalc-dims="1" /></a></em>
  </p>
  
  <p>
    <strong>Fig 17. Repeat-masker repeats report for fosmid14 </strong>
  </p>
</div>